troposphere/settings/local.py.j2

```diff
@@ -17,12 +17,15 @@ API_V2_ROOT = API_SERVER + "/api/v2"
 
 SITE_TITLE= "{{ SITE_TITLE}}"
 SITE_FOOTER= "{{ SITE_FOOTER }}"
+
 {% if THEME_NAME is defined and not THEME_NAME == "default" %}
 THEME_NAME="{{ THEME_NAME }}"
 {% else %}
-THEME_NAME="troposphere_theme"
+THEME_NAME= "troposphere_theme"
```

Comment:
Only reason not to rename would be that CyVerse is now the default, since we are not likely to revert to the old iPlant theme. But I can go either way on this, leaning toward repurposing the `troposphere_theme` name.